opensrp-chw-hf/src/main/assets/json.form/hiv_outcome.json

```diff
@@ -146,6 +228,20 @@
             "test_results_calculation"
           ]
         },
+        {
+          "name": "other_reason_no_action_was_taken",
+          "type": "text_input_edit_text",
+          "properties": {
+            "hint": "Others (Specify)"
+          },
+          "meta_data": {
+            "openmrs_entity": "concept",
+            "openmrs_entity_id": "reason_no_action_was_taken",
+            "openmrs_entity_parent": ""
+          },
+          "required_status": "true:Please specity other reason",
+          "subjects": "reason_no_action_was_taken:map, action_taken_hiv_problems:map"
+        },
```

Comment:
Same here, the requirement is to add other places where the test was conducted, not other reasons no action was taken